.circleci/config.yml

```diff
@@ -0,0 +1,70 @@
+version: 2
+jobs:
+  build:
+    parallelism: 4
+    docker:
+      - image: golang:1.8
+    environment:
+      GOVERSION: "1.8.3"
+      GOPATH:  "/root/.go_workspace"
+      WORKDIR: "/root/.go_workspace/src/k8s.io/helm"
```

Comment:
We won't need `WORKDIR` any more. We can now set the working directory with `working_directory`. See an example [here](https://github.com/thomastaylor312/helm-commandeer/blob/master/.circleci/config.yml#L4)